docs/install.md

```diff
@@ -43,10 +43,11 @@ kubectl create secret -n nim-operator docker-registry ngc-secret \
 ```
 
 ### 5. Install the NIM Operator
-Install the NIM Operator using the Helm chart located in the helm/k8s-nim-operator directory.
+
+Install the NIM Operator using Helm.
 
 ```sh
-helm install nim-operator helm/k8s-nim-operator -n nim-operator
+helm install nim-operator deployments/helm/k8s-nim-operator -n nim-operator
```

Comment:
until the initial version is available in NGC, we can change it to use `--set operator.image.repository=<your-private-repository> --set operator.image.tag=<tag>`.

@ArangoGutierrez should we publish the latest image tag in GH and reference in the helm charts? When we publish to NGC, it will be updated. That way users can try out using helm without any custom builds.